utils/log.py

```diff
@@ -206,7 +206,11 @@ def __init__(self, prefix=None):
 
     def filter(self, record):
         if self.prefix:
-            record.msg = "{}{}".format(self.prefix, record.msg)
+            try:
```

Comment:
@psav this avoid fixing up data that's not broken, we try to do it in good faith first, and if the passed in data is bad we can fix it up

we might want to put a warning out and tell the source of the issue (as it points to subtle Unicode missuses on python2)

however personally i believe we can just mark it as try to remove after porting to python3 and let the issue be